lib/hexo/load_config.js

```diff
@@ -30,10 +30,27 @@ module.exports = async ctx => {
   validateConfig(ctx);
 
   ctx.config_path = configPath;
-
+  // Trim multiple trailing '/'
   config.root = config.root.replace(/\/*$/, '/');
+  // Remove any trailing '/'
   config.url = config.url.replace(/\/+$/, '');
 
+  // Deprecated: config.external_link boolean option will be removed in future
+  if (config.external_link === true) {
+    config.external_link = {
+      enable: true,
+      field: 'site',
+      exclude: []
```

Comment:
`exclude` defaults to an [empty string](https://github.com/hexojs/hexo/blob/5b2808c5df563f8cf40c4bdc9c17e8ca758736ff/lib/hexo/default_config.js#L36). Perhaps can simplify to `config.external_link = defaultConfig.external_link`.

Suggest to update L50 too for consistency.

Reply:
@curbengh  Done.